index.html

```diff
@@ -3381,7 +3381,8 @@ <h2>DID Resolution</h2>
 relating to the results of the <a>DID resolution</a> process which typically
 changes between invocations of the <code>resolve</code> and
 <code>resolveRepresentation</code> functions, as it represents data about the
-resolution process itself. This structure is REQUIRED, and MUST NOT be empty.
+resolution process itself. This structure is REQUIRED, and in the case of an
+error in the resolution process, this MUST NOT be empty.
```

Comment:
I'm concerned about the testability of this statement... it presumes a protocol is in play, but since @peacekeeper is writing these tests, I defer to him to figure out how to test it.